cluster/manifests/aws-efa-device-plugin/rbac.yaml

```diff
@@ -0,0 +1,27 @@
+{{ if eq .Cluster.ConfigItems.aws_efa_device_plugin_enabled "true" }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: aws-efa-k8s-device-plugin
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: aws-efa-k8s-device-plugin
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: aws-efa-k8s-device-plugin
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: aws-efa-k8s-device-plugin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: privileged-psp
```

Comment:
This doesn't work anymore. We need to define the service account in this list: https://github.com/zalando-incubator/kubernetes-on-aws/blob/fae0f905acdd619b18bca11cd9d25cc5b67a3bca/cluster/manifests/01-admission-control/config.yaml#L95-L114

(And remove this role binding which is not needed)